test/IntegrationTests/SmokeTests.cs

```diff
@@ -55,23 +55,26 @@ public async Task SubmitsTraces()
     [Trait("Category", "EndToEnd")]
     public async Task WhenStartupHookIsNotEnabled()
     {
+        SetEnvironmentVariable("DOTNET_STARTUP_HOOKS", null);
 #if NETFRAMEWORK
-        await VerifyTestApplicationInstrumented(enableStartupHook: false);
+        await VerifyTestApplicationInstrumented();
 #else
         // on .NET Core it is required to set DOTNET_STARTUP_HOOKS
-        await VerifyTestApplicationNotInstrumented(enableStartupHook: false);
+        await VerifyTestApplicationNotInstrumented();
 #endif
     }
 
     [Fact]
     [Trait("Category", "EndToEnd")]
     public async Task WhenClrProfilerIsNotEnabled()
     {
+        SetEnvironmentVariable("COR_ENABLE_PROFILING", "0");
+        SetEnvironmentVariable("CORECLR_ENABLE_PROFILING", "0");
```

Comment:
ok to make it explicit it here given the test goal.

Reply:
Should I do something? I am asking because I am not sure 😉

Reply:
no, it is good as it is ;)